packages/uniforms-material/package.json

```diff
@@ -54,7 +54,7 @@
     "eslint-plugin-react": "6.10.0",
     "eslint-plugin-vazco": "1.0.0",
     "jsdom": "9.11.0",
-    "material-ui": "0.16.7",
+    "material-ui": "0.17.0 || 0.16.7",
```

Comment:
...and here, it would be better to keep only one possible version, to make tests deterministic.